ui/.eslintignore

```diff
@@ -0,0 +1 @@
+src/tanagra-api/
```

Comment:
Question: Is this automatic generated or manually add? What kind of things igore in this file?